src/pypromice/process/L1toL2.py

```diff
@@ -72,9 +75,9 @@ def toL2(
     ds = L1.copy(deep=True)                                                    # Reassign dataset
     ds.attrs['level'] = 'L2'
     try:
```

Comment:
At the moment, we might not be able to trace whether the problem lies in the `adjustTime`, `flagNAN` or `adjustData` if an exception is raised. Should we change this so that traceability is better?

Reply:
Guess this should be a separate PR.

Reply:
`logger.exception` will include a traceback which includes line numbers

```
import logging

logger = logging.getLogger(__name__)


def baz():
    raise Exception("baz")

def bar():
    return baz()

def foo():
    try:
        bar()
    except Exception as e:
        logger.exception(e)

foo()
```
Yields
```
Traceback (most recent call last):
  File "<ipython-input-10-97922593f0a9>", line 14, in foo
    bar()
  File "<ipython-input-10-97922593f0a9>", line 10, in bar
    return baz()
  File "<ipython-input-10-97922593f0a9>", line 7, in baz
    raise Exception("baz")
Exception: baz
```